### PR TITLE
(core): automatically update Homebrew core formula with every release

### PR DESCRIPTION
**Pull request by christian-bromann**
I've never used this Github Action so let's pay attention on the next release.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -59,3 +59,13 @@ jobs:
...
         if: env.pre_release == 'false'
         run: gsutil -m cp "gs://runme-artifacts-41eac6/${{ env.version }}/*" gs://runme-artifacts-41eac6/latest
+      - name: Bump Homebrew Formula
+        uses: mislav/bump-homebrew-formula-action@v2
+        # skip prereleases
+        if: ${{ !contains(github.ref, '-') }}
+        with:
+          formula-name: runme
+          homebrew-tap: Homebrew/homebrew-core
+          base-branch: master
```
Comment (mxsdev) [inline .github/workflows/release.yml:69]:
```suggestion
          base-branch: main
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -59,3 +59,13 @@ jobs:
...
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (mxsdev) [inline .github/workflows/release.yml:71]: According to the [docs for this action](https://github.com/mislav/bump-homebrew-formula-action):

> You should enable GITHUB_TOKEN only if the repository that runs this Action is private and if COMMITTER_TOKEN has the public_repo scope only. GITHUB_TOKEN will be used for verifying the SHA256 sum of the downloadable archive for this release.

Apparently we're supposed to create a new personal access token for this purpose, with the `repo` and `workflow` scopes.

**Review by mxsdev — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (christian-bromann) [reply .github/workflows/release.yml:69]: This refers to https://github.com/Homebrew/homebrew-core main branch afaik. I guess we could also just remove this as it is the default value I assume

File: .github/workflows/release.yml (hunk 2 above)
Comment (christian-bromann) [reply .github/workflows/release.yml:71]: Do you think we can use the same token for both?

Comment (christian-bromann) [reply .github/workflows/release.yml:71]: @sourishkrout mind adding a repository secret? I don't have access to that section.

Comment (sourishkrout) [reply .github/workflows/release.yml:71]: You're admin on the repo now

**Comment by christian-bromann** [comment]
Added a `COMMITTER_TOKEN` secret with the right access rights.

**Review by mxsdev — APPROVED** [review]
Let's keep an eye out next release 👍

**Pull request merged**